### PR TITLE
Ongoing PR for next version: NewsML-G2 v2.32

### PR DESCRIPTION
**Pull request by bquinn**

### CODE REVIEW

**Review by sinh8erjones — APPROVED** [review]

**Comment by bquinn** [comment]
Approved at IPTC Standards Committee on 17 May 2023.

**Review by pauliharman — APPROVED** [review]

File: tests/schema_versions/NewsML-G2_2.32-spec-All-Power.xsd
Comment (davecompton) [inline tests/schema_versions/NewsML-G2_2.32-spec-All-Power.xsd:1]: @bquinn :
<xs:attributeGroup name="authorityAttributes"> is in i/ the pre mtg schema at https://iptc.org/std-dev/NewsML-G2/2.32/specification/NewsML-G2_2.32-spec-All-Power.xsd and ii/ NewsML-G2_2.32-spec-All-Power_1.xsd in this PR  ... but I don't see it in this file?

**Review by davecompton — CHANGES_REQUESTED** [review]
Just one comment re <xs:attributeGroup name="authorityAttributes">

File: tests/schema_versions/NewsML-G2_2.32-spec-All-Power.xsd
Comment (bquinn) [reply tests/schema_versions/NewsML-G2_2.32-spec-All-Power.xsd:1]: I had forgotten to push the copy of the schema that sites in the unit tests folder. But the main schema was correct. Thanks for pointing that out.

**Pull request merged**